pkg/mqtrigger/metrics.go

```diff
@@ -45,6 +45,20 @@ var (
 		},
 		[]string{"trigger_name", "trigger_namespace", "topic", "partition"},
 	)
+	triggerStatus = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "fission_mqt_trigger_status",
```

Comment:
Please change this to `fission_mqt_trigger_status` as mqt contains reference to trigger.

Reply:
Done